spec/ddtrace/pipeline/span_filter_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'spec_helper'
+require 'ddtrace/pipeline'
+require 'ddtrace/span'
+
+RSpec.describe Datadog::Pipeline::SpanFilter do
+  def generate_span(name, parent = nil)
+    Datadog::Span.new(nil, name).tap do |span|
+      span.parent = parent
+    end
+  end
+
```

Comment:
You might want to move this method in to a shared file, like Sidekiq does here: https://github.com/DataDog/dd-trace-rb/tree/64ec7458bd9e6fe010382cdd655044a8aca7264f/spec/ddtrace/contrib/sidekiq/support

We normally add a `support/` directory with shared files in it.

Reply:
makes sense, updated